docsite/rst/developing_modules.rst

```diff
@@ -793,6 +793,27 @@ This example allows the stat module to be called with fileinfo, making the follo
     ansible -m fileinfo -a "path=/tmp" localhost
     '''
 
+Deprecating module arguments
+````````````````````````````
+
+Starting in 2.2, you can deprecate module arguments by adding a deprecated='yes' to the arg_spec, e.g. for deprecation is_enabled::
+
+    module = AnsibleModule(
+        argument_spec = dict(
+            name=dict(required=True, aliases=['domain']),
+            force=dict(type='bool', default='no'),
+            is_enabled=dict(type='bool', default='no', deprecated='yes'),
```

Comment:
i would make this an actual boolean, not 'string' yes, this is python code, so True/False, also `default=False` should be the example for booleans.